sdk/keyvault/keyvault-keys/gulpfile.js

```diff
@@ -0,0 +1,13 @@
+const gulp = require("gulp");
```

Comment:
I assume this is copied from Storage, but it's only used because Storage has a unique requirement to generate a .zip bundle which other packages should **not** by copying by default.  For most other packages, the `.tgz` published to npmjs.com is sufficient.